xmodule/mongo_utils.py

```diff
@@ -66,14 +65,6 @@ def connect_to_mongodb(
         connection_params.update({'username': user, 'password': password, 'authSource': db})
 
     mongo_conn = pymongo.MongoClient(**connection_params)
-
-    if proxy:
```

Comment:
Since `proxy` is now unused, could you remove it from the parameters list and the docstring for this method?